src/components/tables/DataTableCell.vue

```diff
@@ -50,14 +50,18 @@ export default {
       return moment(date).format('D MMM YYYY');
     },
     openUserProfile: function(options) {
-      Fliplet.Studio.emit('overlay', {
-        name: 'edit-organization-user',
-        options: {
-          size: 'large',
-          title: 'Edit User',
-          userId: options.userId
-        }
-      });
+      Fliplet.Organizations.get()
```

Comment:
This suggests that when a user profile is to be opened, we're making an API request to fetch the organisation information just to get an organisation ID (that never changes throughout the session).

It introduces:

1. A delay between clicking the user profile and opening up the user profile
1. Unnecessary subsequent API requests, all creating the same delay

Use `Fliplet.Env.get('organizationId')` instead, which gives you a synchronous value of the organisation ID,

Reply:
@tonytlwu Thanks it works much better now. Made new commit with these changes.